src/image/image_raw.rs

```diff
@@ -285,20 +285,22 @@ where
     type Color = CS::Color;
 
     fn pixel(&self, p: Point) -> Option<Self::Color> {
-        if p.x < 0 || p.y < 0 || p.x >= self.size().width as i32 || p.y >= self.size().height as i32
-        {
+        if p.x < 0 || p.y < 0 {
```

Comment:
I don't think you can remove the `p.x` check (or `p.y` for vertical images). Without the check `img.pixel(Point::new(image_witdth, 0))` would warp and return the same as `img.pixel(Point::new(1, 0))`, but it should return `None`.

Reply:
Argh, right you are. I added those checks back in to the unsigned `x` and `y` vars which made the function the same speed as the original code, so I reverted it.

The fastest code is no code, and the most bug-free code is also no code so... should I just delete e-g? I dunno man...